### DISCUSSION TITLE
Box Anemometer

### DISCUSSION THREAD

**Discussion by rsykes-dba** [community/community#101833] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I have installed the MySQL version of this product. Is there a postgres version? Or any other open source supported version.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by rsykes-dba** [reply]
Thanks for your reply. I am already using the MySQL version of this product I mentioned. I am trying to find out if the product is available for postgres or any other open source. It's doesn't mention that in the MySQL download information.

**Reply by rsykes-dba** [reply]
Contributors are there yet I don't see how to contact them. It brings me to github
